src/dev/pages/field/field.ejs

```diff
@@ -0,0 +1,67 @@
+<div class="vert">
+  <div>
+    <h3 class="forge-typography--heading2">Default</h3>
+    <forge-field>
+      <label slot="label">Label</label>
+      <input type="text" value="Content" />
+    </forge-field>
```

Comment:
I noticed a small issue here when I removed the `<label>` element:
```html
<forge-field>
  <input type="text" />
</forge-field>
```

It looks like it's still applying the inset animation to the content when there is no `<label>`.

Reply:
I almost don't think that's a bug. If someone wants to use this without a label they should set `labelPosition` to `none` 🤷‍♂️

But I'll fix it anyway.